README.md

```diff
@@ -269,11 +270,29 @@ such as a `config/` directory.
 If you don't have a configuration file, or want to ignore it if it does exist,
 you can pass `--no-config` instead.
 
-#### `--debug-check`
+#### `--ignore-path`
 
-If you're worried that Prettier will change the correctness of your code, add `--debug-check` to the command.
-This will cause Prettier to print an error message if it detects that code correctness might have changed.
-Note that `--write` cannot be used with `--debug-check`.
+Path to a file containing patterns that describe files to ignore.  By default, prettier looks for `./.prettierignore`.
+
+#### `--list-different`
+
+Another useful flag is `--list-different` (or `-l`) which prints the filenames of files that are different from Prettier formatting. If there are differences the script errors out, which is useful in a CI scenario.
+
+```bash
+prettier --single-quote --list-different "src/**/*.js"
+```
+
+#### `--no-config`
+
+Do not look for a configuration file.  The default settings will be used.
+
+#### `--with-node-modules`
+
+Prettier CLI will ignore files located in `node_modules` directory. To opt-out from this behavior use `--with-node-modules` flag.
+
+#### `--write`
```

Comment:
Wow, crazy we didn't even document write!

Reply:
😊 

I love automation.  Ideally, I'd setup something that prints the CLI options to the README.  It may not be as verbose, but it would ensure all the options are documented in one place.

Reply:
If you want to set this up, that would be awesome.

Reply:
I would move all those explanations as comments in the code and extract them to reproduce the README.

Reply:
I'll take a look

Reply:
We could also merge this as-is and improve things further in another PR.

Reply:
Yeah, doesn't have to be in this PR. The only blocker for merging this is moving the toc extraction out of the build file and make a different command.

Reply:
Looks like there is already some splitting of the options in `prettier.js` so that they can be added to `minimist` and handled in other places (e.g. `booleanOptionNames`).

Would you support a single object that defined `prettier`?  I'm thinking something like:

```js
const prettier = {
  write: {
    type: 'boolean',
    cliDescription: 'Edit the file in-place. (Beware!)',
    docDescription: 'This rewrites all processed files in place.  This is comparable to the `eslint --fix` workflow.'
  }
}
```

This way, there can be a SSOT for defining these things.

Reply:
Ignore my last, github didn't update your replies before I posted.  This PR is good to go then.